ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x__4.13.yaml

```diff
@@ -43,6 +43,31 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: acceptance-lp-interop
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+    test:
+    - as: sbo-acceptance-tests
+      cli: latest
+      commands: |
+        export CATSRC_NAME=redhat-operators
+        export OPERATOR_CHANNEL=stable
+        export OPERATOR_PACKAGE=rh-service-binding-operator
+        export SKIP_REGISTRY_LOGIN=true
+        ./install.sh
+        export TEST_ACCEPTANCE_START_SBO=remote
+        export EXTRA_BEHAVE_ARGS="--tags=~@crdv1beta1 --tags=~@olm-descriptors --tags=~@upgrade-with-olm --tags=~@disable-openshift-4.13 --tags=~@disable-openshift-4.12 --tags=~@disable-openshift-4.8+ --tags=~@disable-openshift-4.9+ --tags=~@disable-openshift-4.10+ --tags=~@disable-openshift-4.11+ --tags=~@disable-openshift-4.12+ --tags=~@disable-openshift-4.13+"
+        make -k VERBOSE=2 -o registry-login test-acceptance test-acceptance-artifacts
+        rename TESTS junit_TESTS /logs/artifacts/acceptance-tests/TESTS*.xml
```

Comment:
Wondering if its cleaner to move these commands to a seperate file.

Reply:
I believe it would, but as far as maintenance goes I think this might be a problem. The SBO team made the decision to work within OpenShift CI before we did and the approach to use dockerfile literals for image creation and commands straight in the config file was made.

If we were to add just what our team needs to test as a separate ref, when it comes time to generate the next file for the next layered product or OCP release it will be more difficult for the SBO team as they'll need to change more things. Also the team maintaining these files are not the ones who created it so I didn't want to overwhelm them.

Reply:
alright, this works fine then